ticket-booker/src/components/Header/HeaderButton.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  ButtonContainer,
+  LeftContainerContent,
+  ButtonGridContainer,
+} from './HeaderButton.style';
+import { IconContainer } from './HeaderButton.style';
+import TicketsIcon from '../../Assets/TicketsIcon.png';
+import { Text } from '../Text/Text';
+export const HeaderButton = () => {
+  return (
+    <ButtonGridContainer>
+      <ButtonContainer>
+        <LeftContainerContent>
+          <IconContainer src={TicketsIcon} />
+          <Text margin={'0 0 0 10px'}>All Tickets</Text>
+        </LeftContainerContent>
+        <Text margin={'0 10px 0 0'}>6</Text>
+      </ButtonContainer>
+      <ButtonContainer>
+        <LeftContainerContent>
+          <IconContainer src={TicketsIcon} />
+          <Text margin={'0 0 0 10px'}>Travels</Text>
+        </LeftContainerContent>
+        <Text margin={'0 10px 0 0'}>6</Text>
+      </ButtonContainer>
+      <ButtonContainer>
+        <LeftContainerContent>
+          <IconContainer src={TicketsIcon} />
+          <Text margin={'0 0 0 10px'}>Movies</Text>
+        </LeftContainerContent>
+        <Text margin={'0 10px 0 0'}>6</Text>
+      </ButtonContainer>
+      <ButtonContainer>
+        <LeftContainerContent>
+          <IconContainer src={TicketsIcon} />
+          <Text margin={'0 0 0 10px'}>Concerts</Text>
+        </LeftContainerContent>
+        <Text margin={'0 10px 0 0'}>6</Text>
+      </ButtonContainer>
```

Comment:
each ButtonContainer looks similar.
Maybe you can create a new component and use map to loop an array to return them?
```
[{text1: "All trickets", text2: "6"}, ..].map(({text1, text2}) => {
<NewComponent text1="text1", text2="text2">
})

```

Reply:
Will implement this idea in future sprint